rebasehelper/utils.py

```diff
@@ -892,6 +892,18 @@ def command_commit(self, message=None, amend=False):
 
         return self._call_git_command(cmd)
 
+    def command_format_patch(self, patchname="changes.patch", directory="./"):
```

Comment:
Why are you using double quotes here and single quotes below? If there is no explicit reason, I think you should choose one or the other for the sake of consistency.